app/src/components/governance/PageProposal.vue

```diff
@@ -30,20 +30,27 @@
       />
 
       <ModalDeposit
-        v-if="parameters.depositDenom"
+        v-if="status.value === governanceStatusEnum.DEPOSITING"
         ref="modalDeposit"
-        :proposal-id="proposalId"
+        :proposal-id="
+          currentNetwork.network_type === `polkadot`
+            ? proposal.proposalId
+            : proposalId
+        "
         :proposal-title="proposal.title || ''"
-        :denom="parameters.depositDenom"
+        :denom="parameters.depositDenom || currentNetwork.stakingDenom"
+        :number-of-seconds="
+          isPolkadotDemocracy ? Number(proposal.detailedVotes.votesSum) : 0
+        "
         @success="() => afterDeposit()"
       />
-      <ModalVoteSubstrate
+      <ModalVotePolkadot
```

Comment:
why switch from substrate to polkadot?

Reply:
Discussing in another PR with Fabo about this, we agreed that in the end it is more simple to just talk about `Polkadot` and forget about `Substrate`